src/mavsdk/core/flight_mode.h

```diff
@@ -7,6 +7,9 @@ namespace mavsdk {
 
 enum class FlightMode {
     Unknown,
+    FBWA,
```

Comment:
```suggestion
    Fbwa,
```

Reply:
And I'd probably add the new modes at the bottom rather than the top, so we don't change the existing indices.